src/redux/actions/pokemonsActions.js

```diff
@@ -0,0 +1,92 @@
+
+
+export const FETCH_POKEMONS_REQUEST = 'FETCH_POSTS_REQUEST'
+export const FETCH_POKEMONS_SUCCESS = 'FETCH_POSTS_SUCCESS'
+export const FETCH_POKEMONS_SUCCESS2 = 'FETCH_POSTS_SUCCESS2'
+export const FETCH_POKEMONS_ERROR = 'FETCH_POSTS_ERROR'
+export const FILTER_POKEMONS = 'FILTER_POKEMONS'
+export const TRIGEER_FETCH = 'TRIGEER_FETCH'
+export const ADD_SELECTED_POKEMON = 'ADD_SELECTED_POKEMON'
+export const CLEAN_SELECTED_POKEMON = 'CLEAN_SELECTED_POKEMON'
+export const SET_SHOW_TOAST = 'SET_SHOW_TOAST'
+
+
+export const fetchPokemons = (counter) => (dispatch) => {
+  const url = `https://pokeapi.co/api/v2/pokemon?offset=${counter}&limit=20`
```

Comment:
the first part of this url must be in a different file, centralized, so it can be used in multiple places